test/Translator/TranslatorTest.php

```diff
@@ -386,4 +386,20 @@ public function testListenerOnNoMessagesLoadedEventCanReturnTextDomainObject()
         $this->assertNull($doNotTrigger);
         $this->assertEquals('BOOYAH', $result);
     }
+
+    public function testGetAllMessagesLoadedInTranslator()
```

Comment:
I would say the only relevant test here is a) check if the result is an array when you supply it with the translation file and b) check if the size of the array equals the count of `$allMessages`.

Also, what happens when the text domain is not found or the locale does not exist? What is the return type and can you test those two cases as well?